### DISCUSSION TITLE
Do you have m3u8 link for nicktoons africa?

### DISCUSSION THREAD

**Discussion by dantheman121** [iptv-org/community#419] [Q&A]
I would like to watch Transformers: Rescue bots

**Comment by Sphinxroot** [comment]
https://www.youtube.com/results?search_query=transformers+rescue+bots&sp=EgJAAQ%253D%253D

**Reply by dantheman121** [reply]
No

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel
